geonode_mapstore_client/client/js/plugins/ActionNavbar.jsx

```diff
@@ -22,15 +22,29 @@ function checkResourcePerms(menuItem, resourcePerms) {
     }
     return true;
 }
+
 function ActionNavbarPlugin({
     items,
     leftMenuItems,
     rightMenuItems,
     resourcePerms
 }, context) {
 
+
     const { loadedPlugins } = context;
     const configuredItems = usePluginItems({ items, loadedPlugins });
+
+
+    const leftMenuItemsPlugins = reduceArrayRecursive(leftMenuItems, (item) => {
+        configuredItems.find(plugin => {
+            if ( item.type === 'plugin' && plugin.name === item.name ) {
+                item.Component = plugin?.Component;
+            }
+        });
+        return (item);
+    });
```

Comment:
If we decide that all the configuredItems should be included these two list must be removed: `leftMenuConfiguredItems` and `rightMenuConfiguredItems`.
In addition we could call the configuration `menuItems` instead of leftMenuItems and remove the prop rightMenuItems.

```js
const leftMenuItemsPlugins = reduceArrayRecursive(menuItems, (item) => {
    configuredItems.find(plugin => {
        if ( item.type === 'plugin' && plugin.name === item.name ) {
            item.Component = plugin?.Component;
        }
    });
    return (item);
});

const leftItems = reduceArrayRecursive(leftMenuItemsPlugins, menuItem => checkResourcePerms(menuItem, resourcePerms);

return (
    <ActionNavbar
        leftItems={leftItems}
    />
);
```

Reply:
If I understand, in the actionnavbar plugin, will we have links aligned to the left?